action.yml

```diff
@@ -19,13 +22,15 @@ runs:
           actionlint ${{ inputs.actionlint-version }}
 
     - name: Configure actionlint
-      shell: bash
+      shell: python
       run: |
-        cat << EOF > .github/actionlint.yaml
+        labels = "${{ inputs.self-hosted-runners }}".split(",")
+        config = f"""\
         self-hosted-runner:
-          labels:
-            - aws-gpu-runner
-        EOF
+          labels: {labels}
+        """
+        with open(".github/actionlint.yaml", "w") as fd:
+            fd.write(config)
```

Comment:
What if the file already exists?
What if the file already has the `self-hosted-runner.labels` key?
What if `self-hosted-runner.labels` is already filled?

Reply:
I've rewritten the Python script to account for that